### DISCUSSION TITLE
Possible to only track URLs, not any other data?

### DISCUSSION THREAD

**Discussion by mlissner** [plausible/analytics#2411] [Anything Else]
I want to use plausible to only track the URLs that are visited, not referer, OS, device, region, etc. 

Can that be configured somehow? 

(here's the reason, if you're curious: https://github.com/freelawproject/recap/issues/246)

**Comment by metmarkosaric** [comment]
hi @mlissner! we don't have any setting that allows you to disable tracking but perhaps something like our events API would be useful to explore: https://plausible.io/docs/events-api

**Comment by mlissner** [comment]
Cool, thanks, makes sense.

**Comment by susanthenerd** [comment]
Adding a way for website owners to disable what is not needed would be good for both the infrastructure and for privacy(less data is always better).
